### PR TITLE
Feature/cmp 629 sdk support for edit items

### PR DESCRIPTION
**Pull request by PeterSkoda**
Added EditLinkBuilder for building URL to edit page of content items

### CODE REVIEW

File: KenticoCloud.ContentManagement.Helpers.Tests/EditLinkBuilderTests.cs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        {
+            var options = new ContentManagementHelpersOptions
+            {
+                ProjectId = "14dc0cf8-6cc1-4f20-8e2e-0b5edea89e43"
+            };
+
+            var underTest = new EditLinkBuilder(options);
+            var result = underTest.BuildEditItemUrl(_language, _itemId);
+
+            var expected = "https://app.kenticocloud.com/goto/edit-item/item/1cdaa8ef-cb2b-4f82-82e9-45467b2e01b9/variant-codename/some-Language-Codename/project/14dc0cf8-6cc1-4f20-8e2e-0b5edea89e43";
```
Comment (kenticomartinh) [inline KenticoCloud.ContentManagement.Helpers.Tests/EditLinkBuilderTests.cs]: Is there any specific reason to have "item" twice in the URL?
It is also weird to have project ID at the end, it should be ordered by the granularity of it, shouldn't it?

File: KenticoCloud.ContentManagement.Helpers/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("KenticoCloud.ContentManagement.Helpers")]
+[assembly: AssemblyDescription("Kentico Cloud Content Management SDK Helpers")]
+[assembly: AssemblyCompany("Kentico Software")]
+[assembly: AssemblyProduct("Kentico Cloud")]
+[assembly: AssemblyCopyright("© 2017 Kentico Software. All rights reserved.")]
```
Comment (kenticomartinh) [inline KenticoCloud.ContentManagement.Helpers/Properties/AssemblyInfo.cs]: 2018?

Comment (PeterSkoda) [reply KenticoCloud.ContentManagement.Helpers/Properties/AssemblyInfo.cs]: Copy-paste :-D sorry, I will fix it

File: KenticoCloud.ContentManagement.Helpers.Tests/EditLinkBuilderTests.cs (hunk 1 above)
Comment (PeterSkoda) [reply KenticoCloud.ContentManagement.Helpers.Tests/EditLinkBuilderTests.cs]: 1) to have word "item" in path is necessary for future compatibility
2) order of project - variant - item is fixed now

File: KenticoCloud.ContentManagement.Helpers/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (PeterSkoda) [reply KenticoCloud.ContentManagement.Helpers/Properties/AssemblyInfo.cs]: Fixed

**Pull request merged**